src/spells5.cc

```diff
@@ -919,30 +919,6 @@ void school_spells_init()
 	}
 
 	{
-		spell_type *spell = spell_new(&DISARM, "Disarm");
-		spell_type_describe(spell, "Destroys doors and traps");
```

Comment:
Like with anti-magic characters I'm wondering about the door-destruction bit here...